core/vm/evm.go

```diff
@@ -104,6 +104,10 @@ type EVM struct {
 	// abort is used to abort the EVM calling operations
 	// NOTE: must be set atomically
 	abort int32
+	// callGasTemp holds the gas available for the current call. This is needed because the
+	// available gas is calculated in gasCall* according to the 63/64 rule and later
+	// applied in opCall*.
+	callGasTemp uint64
```

Comment:
Perhaps `callGasReal` (as in the real value, not the requested amount?). Temp seems a bit strange.

Reply:
I chose "temp" because the field holds the gas value temporarily, in-between calls to `gasCall*` and `opCall*`.